tests/testthat/test-generate_cre_dataset.R

```diff
@@ -1,19 +1,19 @@
 test_that("Synthetic Data Generated Correctly", {
   # Incorrect n input
-  expect_error(generate_cre_dataset(n = "test", rho = 0, n_rules = 2, effect_size = 0.5, binary = FALSE, seed = 2021))
+  suppressWarnings(expect_error(generate_cre_dataset(n = "test", rho = 0, n_rules = 2, effect_size = 0.5, binary = FALSE, seed = 2021)))
 
   # Incorrect rho input
-  expect_error(generate_cre_dataset(n = 1000, rho = "test", n_rules = 2, effect_size = 0.5, binary = FALSE, seed = 2021))
+  suppressWarnings(expect_error(generate_cre_dataset(n = 1000, rho = "test", n_rules = 2, effect_size = 0.5, binary = FALSE, seed = 2021)))
```

Comment:
Why are you suppressing Warnings?

Reply:
Just to simplify the output of testthat, but I can remove them!